app/views/buses/arrivals.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Buses</h1>
```

Comment:
Should we say if it's the postcode or bus stop? Since both of the endpoints use the same view. Would it be better to have 2 templates for each and then use this as a common thing, just use different headings for different endpoints?

Reply:
I've added the names of the stops and their directions since that's probably what a user wants to see rather than a stop id